onnx2kerastl/converter.py

```diff
@@ -127,9 +127,17 @@ def onnx_to_keras(onnx_model, input_names, name_policy=None, verbose=True, chang
     # Convert every operation separable
     node_names = []
     for node_index, node in enumerate(onnx_nodes):
+        if node.op_type == 'If':
```

Comment:
the if gets constant boolean? so why onnx not just remove it?

Reply:
Currently, the way we deal with the `IF` node is to parse it and remove it from the graph. This fixes the computational graph but is the simplest solution at this point. In case a need arises we can support the IF node in a dynamic manner but it would require a heavier change to the flow.